### ISSUE TITLE
Change Locale by Application Preference

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#40]
```
I did some experimenting with a Locale Preference per Application, so that the 
user can choose a Locale other then the system default. This patch is the 
result.
It works fine on the emulator, although a restart of the MainScreen Activity 
and the Service is required for the changes to take effect.

Feel free to comment and use/discard the patch.
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 8 Jan 2011 at 2:17

Attachments:
- [locale_pref.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-40/comment-0/locale_pref.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks I'll will try to auto reload view on locale change.
Don't forget to update xlsx file for translations.
```

Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 4:11
- Changed title: **Change Locale by Application Preference**
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 5:58
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 11:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.7
```

Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 11:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
